src/main/resources/application.properties

```diff
@@ -287,4 +288,7 @@ reference.data.cache.ttl.in.days=5
 # Refresh cache once everyday at 4am
 reference.data.cache.refresh.rate.cron=0 0 4 * * ?
 
+definition-store.retry.maxAttempts=${DEFINITION_STORE_RETRY_MAX_ATTEMPTS:2}
+definition-store.retry.maxDelay=${DEFINITION_STORE_RETRY_MAX_DELAY:1000}
+
 
```

Comment:
do we need to reference these HTTP_CLIENT_CONNECTION_DEFINITION_STORE_TIMEOUT, DEFINITION_STORE_RETRY_MAX_ATTEMPTS, DEFINITION_STORE_RETRY_MAX_DELAY in either the values.yaml or flux config?

Reply:
These are the default values and we will use flux when we need to change them.

Reply:
including HTTP_CLIENT_CONNECTION_DEFINITION_STORE_TIMEOUT